<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>X-Sniffer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>721</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止拦截</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>网络层分析</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清除数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TransButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>传输层分析</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tcpopt">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP协议</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="udpopt">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>60</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP协议</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="icmpopt">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ICMP协议</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="igmpopt">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>IGMP协议</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="arpopt">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ARP协议</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="filter_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始过滤</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="otheropt">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>其他协议</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_fenxi">
    <property name="title">
     <string>分析</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>拦截</string>
    </property>
    <addaction name="action_start"/>
    <addaction name="action_stop"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_fenxi"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>网络层分析</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>传输层分析</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_start">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
